acceptance/tests/agent/last_run_summary_report.rb

```diff
@@ -0,0 +1,108 @@
+test_name "The 'last_run_summary.yaml' report has the right location and permissions" do
+  tag 'risk:medium',
+      'audit:high'
```

Comment:
We should only have `audit:high` here, as `risk:medium` is redundant.

Reply:
Actually I don't exactly know what these tags mean. In CI we run `risk:high` and `audit:high` so I always assumed `risk` and `audit` were interchangeable.

Reply:
You are free to ignore me if I'm saying dumb things.